.github/workflows/test.yml

```diff
@@ -19,36 +19,20 @@ jobs:
       fail-fast: true
       matrix:
         rails-version:
+          - "7.1"
           - "7.2"
```

Comment:
I think we can just make this `"7.0"` for now. Since we (now) know Solidus doesn't support 7.1+ until v4.3 and we aren't testing any of those versions yet.